lib/pages/Rides.dart

```diff
@@ -322,7 +358,13 @@ class RideGroup extends StatelessWidget {
               child: RideGroupTitle(title, rides.length),
             );
           }
-          Widget w = RideCard(rides[index]);
+          Widget w = Opacity(
+              opacity: interactive ? 1 : 0.5,
+              child: RideCard(rides[index])
+          );
+          if (!interactive) {
+            w = IgnorePointer(child: w);
```

Comment:
What is the purpose of using an IgnorePointer?

Reply:
The ride cards are usually tappable to start the ride flow, but shouldn't be in the schedule view, so the IgnorePointer makes them not tappable